admin.html

```diff
@@ -7,11 +7,13 @@
     <title>Admin</title>
 </head>
 <body>
-    <div class="full-container">
+    <button id="full-button" class="full-button">
         Mark as full
-     </div>
-     <div class="vacant-container">
+     </button>
+     <button id="vacant-button" class="vacant-button">
         Mark as vacant
-    </div>
+    </button>
+
```

Comment:
The UX will be better if the user can see the currently set status in this page without needing to visit the other page.